### ISSUE TITLE
0.2.12 > 0.2.13 breaking change (extraSchemas)

### ISSUE THREAD

**Issue by dodas** [Urigo/graphql-modules#226]
Hi!
I updated from 0.12.12 to 0.2.13 and my app suddenly doesn't work anymore.
I am using typescript and this is the error being outputted:
```
/Users/dodas/dev/fakture/packages/server/node_modules/@graphql-modules/core/dist/graphql-module.js:478
            finally { if (e_2) throw e_2.error; }
                               ^

TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
    at Object.__values (/Users/dodas/dev/fakture/packages/server/node_modules/tslib/tslib.js:144:50)
    at GraphQLModule.buildSchemaAndInjector (/Users/dodas/dev/fakture/packages/server/node_modules/@graphql-modules/core/dist/graphql-module.js:459:55)
    at GraphQLModule.get [as resolvers] (/Users/dodas/dev/fakture/packages/server/node_modules/@graphql-modules/core/dist/graphql-module.js:168:22)
    at Module../src/graphql/index.ts (/Users/dodas/dev/fakture/packages/server/dist/index.js:4938:5)
    at __webpack_require__ (/Users/dodas/dev/fakture/packages/server/dist/index.js:21:30)
    at Module../src/pdf-renderer/index.ts (/Users/dodas/dev/fakture/packages/server/dist/index.js:7370:66)
    at __webpack_require__ (/Users/dodas/dev/fakture/packages/server/dist/index.js:21:30)
    at Module../src/index.ts (/Users/dodas/dev/fakture/packages/server/dist/index.js:7014:71)
    at __webpack_require__ (/Users/dodas/dev/fakture/packages/server/dist/index.js:21:30)
    at Object.0 (/Users/dodas/dev/fakture/packages/server/dist/index.js:7686:18)
```

it seems to have something to do with the new `extraSchemas` option, which I am not using.
any ideas?

thanks.

**Comment by ardatan** [comment]
Could you update to latest version 0.2.17?

**Comment by dodas** [comment]
That's what I had initially - same result.
I just downgraded to 0.2.12 and then to 0.2.13 to find out where this breaking change was introduced.

**Comment by ardatan** [comment]
Could you reproduce a repo for it?

**Comment by dodas** [comment]
Managed to reproduce it: [graphql-modules-bug-reproduce](https://github.com/dodas/graphql-modules-bug-reproduce).
Just clone it, run `npm install` and `npm run start`.

It looks like reading `resolvers` and `typeDefs` of module together does trigger this error.
Is this usage correct? Typescript does hint that such properties exists.

**Comment by ardatan** [comment]
Ok, I will check it out. So, in the meanwhile you can use that version or just use the schema exported from the module
```
const { schema }  = yourmodule;
```

**Comment by ardatan** [comment]
Sure.
Could you try if it works with these canary versions?
```
 - @graphql-modules/core@0.3.0-alpha.d74cc126
 - @graphql-modules/di@0.3.0-alpha.d74cc126
 - @graphql-modules/epoxy@0.3.0-alpha.d74cc126
 - @graphql-modules/sonar@0.3.0-alpha.d74cc126
 - @graphql-modules/utils@0.3.0-alpha.d74cc126
```

**Comment by dodas** [comment]
Yes, it works!

**Comment by ardatan** [comment]
Available in 0.3.0
